src/burnchains/burnchain.rs

```diff
@@ -891,8 +891,20 @@ impl Burnchain {
 
         let sortition_tip = SortitionDB::get_canonical_sortition_tip(db.conn())?;
 
-        db.evaluate_sortition(&header, blockstack_txs, burnchain, &sortition_tip, None)
-            .map(|(snapshot, transition, _)| (snapshot, transition))
+        let (parent_snapshot, parent_sort_id, reward_set_info) =
+            db.get_sortition_info(&header, burnchain, &sortition_tip, None)?;
+
+        // The block is not broadcast to the event observer here since this is a legacy function.
```

Comment:
Updated the comment here -- this method is deprecated and invoked only in the defunct helium run-loop and some test methods. Because of that, we never broadcasted burn block info to event observers here, and that doesn't change in this PR.